source/common/router/scoped_rds.cc

```diff
@@ -181,7 +188,12 @@ bool ScopedRdsConfigSubscription::addOrUpdateScopes(
       ENVOY_LOG(debug, "srds: add/update scoped_route '{}', version: {}",
                 scoped_route_info->scopeName(), version_info);
     } catch (const EnvoyException& e) {
-      exception_msgs.emplace_back(absl::StrCat("", e.what()));
+      ENVOY_LOG_MISC(warn, "add or update scope {} failed {} ", resource.name(), e.what());
```

Comment:
Would it make sense to truncate the error message here, rather than do it when throwing an exception?